pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion = "2.1.0-dev"
+	bundleVersion = "2.1.0"
```

Comment:
2.1.0 has already been released, this cannot be 2.1.0 anymore - either next patch or minor

Reply:
I updated it to 2.1.1